packages/components/src/Banner/Banner.mdx

```diff
@@ -66,23 +118,23 @@ within a `Banner`. If you require an action, use the `primaryActions` prop. The
 <Playground>
   <Content>
     <Banner
-      type="success"
+      type="notice"
       primaryAction={{
-        label: "Poke Me",
+        label: "View Plans",
         onClick: () => alert("💥💥SMASH💥💥")
```

Comment:
I approve of the de-Marvelification of the sandbox examples - can we change the alert contents for this and line 134 as well? Maybe something like "Plans" and "Refreshing"...